test/blackbox-tests/test-cases/pkg/opam-var/opam-var-global.t

```diff
@@ -0,0 +1,35 @@
+  $ . ./opam-var-helpers.sh
+
+Here we test the translation and implementation of global opam variables. OS specific
+variables can be found in `opam-var-os.t`.
+
+# jobs
+Should always be 1
+  $ test jobs 
+  (build
+   (run echo %{jobs}))
+  File "dune.lock/testpkg.pkg", line 4, characters 11-18:
+  4 |  (run echo %{jobs}))
+                 ^^^^^^^
+  Error: Unknown variable %{jobs}
+  [1]
+
+# make
+There is a difference here as dune returns the absolute directory to make whereas
+opam returns `make`.
+  $ BUILD_PATH_PREFIX_MAP="MAKE_DIR=$(dirname $(which make)):$BUILD_PATH_PREFIX_MAP"
+  > test make 
+  (build
+   (run echo %{make}))
+  MAKE_DIR/make
+
+# user group
```

Comment:
This is actually tied to the switch config and is created from scratch for every new opam switch. This makes it tricky to get from our `opam var` trick that we used for os therefore we use standard posix methods of getting this information.